### ISSUE TITLE
Not working with Android + Firefox

### ISSUE THREAD

**Issue by heartog** [mun3/F-ckAvgleAntiAdblock#17]
Followed all the steps, installed ubo, added the filters, installed tampermonkey and the script.

When trying to play the video, it would detect i'm using an adblocker. 

Script ver: 0.11.27

**Comment by mun3** [comment]
Can you show me the uBO filters you're using?

**Comment by heartog** [comment]
I manage to solve the issue by using "desktop mode" in Firefox. The only filters I'm using is the ones shown in the readme

**Comment by mun3** [comment]
All 3 filters?
```
avgle.com###player_3x2_container
@@||avgle.com/avideos.php
avgle.com#@#script:inject(abort-on-property-read.js, ExoDetector)
```

**Comment by heartog** [comment]
Yes, using those 3 filters w/ "mobile mode" would not work, but would work with "desktop mode"

**Comment by ghajini** [comment]
It does work..... requires 2-4 clicks/double tap on play button..... overlay is a issue?

**Comment by heartog** [comment]
it didn't before, the last update fixed it

**Comment by ghajini** [comment]
Not working

**Comment by hoonkai** [comment]
I believe maintenance is suspended until donation target is reached https://github.com/mun3/F-ckAvgleAntiAdblock/blob/master/README.md

**Comment by ghajini** [comment]
Ok then ,no one will use this 
.....

**Comment by Yoposoc** [comment]
doesn't work. terrible that the devs won't work on it until they reach a donation target.
